x-pack/plugins/security_solution/server/lib/detection_engine/rules/utils.ts

```diff
@@ -213,24 +216,61 @@ export const transformToAlertThrottle = (throttle: string | null | undefined): s
   }
 };
 
+/**
+ * Given a set of actions from an "alerting" Saved Object (SO) this will transform it into a "security_solution" alert action.
+ * If this detects any legacy rule actions it will transform it. If both are sent in which is not typical but possible due to
+ * the split nature of the API's this will prefer the usage of the non-legacy version. Eventually the "legacyRuleActions" should
+ * be removed.
+ * @param alertAction The alert action form a "alerting" Saved Object (SO).
+ * @param legacyRuleActions Legacy "side car" rule actions that if it detects it being passed it in will transform using it.
+ * @returns The actions of the FullResponseSchema
+ */
+export const transformActions = (
+  alertAction: AlertAction[] | undefined,
+  legacyRuleActions: LegacyRuleActions | null | undefined
+): FullResponseSchema['actions'] => {
+  if (alertAction != null && alertAction.length !== 0) {
+    return alertAction.map((action) => ({
+      group: action.group,
```

Comment:
Frank, isn't this just `transformAlertToRuleAction`? Can you change this to use that one instead?

Reply:
Yeah it is, changing it out. Thanks!